From f75adfde21d763300100356a6b2b7c261cb65b23 Mon Sep 17 00:00:00 2001
From: Julian Bouzas <julian.bouzas@collabora.com>
Date: Thu, 21 Aug 2025 18:09:20 -0400
Subject: [PATCH 5/5] state-routes.lua: Don't save again the route when
 restoring it

There is no need to save the route again after it has been restored because its
value has not changed.
---
 src/scripts/device/state-routes.lua | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/scripts/device/state-routes.lua b/src/scripts/device/state-routes.lua
index 68617ebb..0470493c 100644
--- a/src/scripts/device/state-routes.lua
+++ b/src/scripts/device/state-routes.lua
@@ -79,8 +79,6 @@ find_stored_routes_hook = SimpleEventHook {
           route_info = nil
         else
           log:info (device, "found stored route: " .. route_info.name)
-          -- make sure we save it again
-          route_info.save = true
         end
       end
 
-- 
2.50.1

